src/templates/mgc/auth.yaml

```
@@ -0,0 +1,8 @@
+{{range $remote_name, $remote := .remotes}}
+  {{ if $remote.mgc }}
+    {{ if eq $remote_name ($.current_remote | toString) }}
```

nao precisa do `| toString` aqui
```suggestion
    {{ if eq $remote_name ($.current_remote) }}
```